man/run0.xml

```diff
@@ -219,6 +219,35 @@
     non-zero return value will be returned.</para>
   </refsect1>
 
+  <refsect1>
+    <title>Environment Variables</title>
+
+    <para>As with <literal>systemd-run</literal>, the session will inherit the system
+    environment from systemd. In addition, the following environment variables will be set:</para>
+
+    <variablelist>
+      <varlistentry>
+        <term>TERM</term>
+        <listitem><para>Forwarded from the caller of run0. Can be overriden with <option>--setenv</option></para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term>SUDO_USER</term>
+        <listitem><para>Set to the username of the originating user.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term>SUDO_UID</term>
+        <listitem><para>Set to the user id of the originating user.</para></listitem>
```

Comment:
people understand "user id" sometimes as the string. Hence please say "numeric UNIX user ID" here. To avoid any confusion